src/diffusers/models/attention.py

```diff
@@ -441,3 +441,106 @@ def forward(self, x, emb):
         x = F.group_norm(x, self.num_groups, eps=self.eps)
         x = x * (1 + scale) + shift
         return x
+
+
+class BasicSparseTransformerBlock(nn.Module):
```

Comment:
```suggestion
class BasicSparse3DTransformerBlock(nn.Module):
```

Reply:
As this is quite specific to the `transformer_3d.py` architecture, can we move it into `transformer_3d.py` for now? The `attention.py` file is mostly used only for highly generic attention functionalities

Reply:
Done